src/core/event/index.js

```diff
@@ -53,7 +53,20 @@ export function Events(Base) {
     #enableScrollEvent = true;
     #coverHeight = 0;
 
-    #scrollTo(el, offset = 0) {
+    #delayScrollInterval;
+    async #scrollTo(el, path, offset = 0) {
+      if (document.readyState !== 'complete') {
+        clearInterval(this.#delayScrollInterval);
+        await new Promise(resolve => {
+          this.#delayScrollInterval = setInterval(() => {
+            if (document.readyState === 'complete') {
+              clearInterval(this.#delayScrollInterval);
+              resolve();
+            }
+          }, 100);
+        });
+      }
```

Comment:
Is this reliable? Does the page go into non-complete ready state on each markdown fetch/render? If not, perhaps we should instead track images (etc) ourself, and wait for `load` event on all of them. Need to check this out.